src/it/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCliComponentTest.java

```diff
@@ -151,16 +163,19 @@ public void checkCreateComponentAndCreateAndDeleteSecureURL() throws IOException
     }
 
     @Test
-    @Ignore("not yet supported by odo")
     public void checkCreateComponentAndLinkService() throws IOException, InterruptedException {
         String project = PROJECT_PREFIX + random.nextInt();
         String application = APPLICATION_PREFIX + random.nextInt();
         String component = COMPONENT_PREFIX + random.nextInt();
         String service = SERVICE_PREFIX + random.nextInt();
         try {
-            createComponent(project, application, component, push);
-            odo.createService(project, application, "postgresql-persistent", "default", service);
-            odo.link(project, application, component, COMPONENT_PATH, service, null);
+            createS2iComponent(project, application, component, push);
```

Comment:
test method name should be renamed as well

Reply:
i created a new set of parameters to test both S2i and Devfile components.